controllers/depresolver/depresolver.go

```diff
@@ -14,17 +13,58 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-//Config holds operator configuration
+// EdgeDNSType specifies to which edge DNS is k8gb connecting
+type EdgeDNSType int
+
+const (
+	// DNSTypeNoEdgeDNS is default DNSType. Is used during integration testing when no infoblox or route53 exists.
```

Comment:
```suggestion
	// DNSTypeNoEdgeDNS is default DNSType. Is used during integration testing when no edgeDNS provider exists.
```
Let's relax the explicit names here as we are about to add more providers in future